src/components/map/MapService.js

```diff
@@ -673,9 +670,9 @@ goog.require('ga_urlutils_service');
               tileHeight: params.tileSize,
               hasAlphaChannel: (format == 'png'),
               availableLevels: window.imageryAvailableLevels,
-              // Experimental: restrict all rasters to terrain availability
-              metadataUrl: getTerrainTileUrl(
-                  gaGlobalOptions.defaultTerrain, terrainTimestamp) + '/'
+              // Experimental: restrict all rasters from 0 - 17 to terrain
+              // availability and 18 to Swiss bbox
+              metadataUrl: '//3d.geo.admin.ch/imagery/'
```

Comment:
All urls are in the [index.mako.html](https://github.com/geoadmin/mf-geoadmin3/blob/master/src/index.mako.html#L852) , can you move it there please.